sdk/search/Microsoft.Azure.Search.Service/src/Customizations/Indexes/Models/StandardTokenizer.cs

```diff
@@ -1,60 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Let's hold off on deleting the "obsolete" analysis components for now. We should first check that they're no longer part of any index, but I'm not sure we have the telemetry yet to answer this question. It isn't a priority right now, so for now just keep them around.

Reply:
Got it, I'll remove other methods/parameters...